libs/navigation/build.mjs

```diff
@@ -0,0 +1,20 @@
+import * as esbuild from 'esbuild'; // eslint-disable-line
+import fs from 'node:fs';
+
+fs.rmSync('./dist/', { recursive: true, force: true });
+
+await esbuild.build({
+  entryPoints: ['navigation.js'],
+  bundle: true,
+  splitting: true,
+  format: 'esm',
+  minify: true,
```

Comment:
Do we want to minify the bundle, does it make any sense? Personally I love to debug in chrome directly using overwrites, that would make this a bit more of an entrance barrier for other folks to come and help out / troubleshoot.

Reply:
Yeah it can save us, overall, a fair few kilobytes (5-10kb), and we can look into generating sourcemaps to help with debugging, and so on. (Source maps would be the preferred way of doing this as well since the chunking will mess up our organization of the code anyway)

Reply:
Since we're not committing the built files to the repo anymore. We won't need to worry about inflating the number of files with sourcemaps.